docs/hipify-clang.rst

```diff
@@ -0,0 +1,809 @@
+      `17.0.6 <https://github.com/llvm/llvm-project/releases/tag/llvmorg-17.0.6>`_\*\*,
+      `18.1.0 <https://github.com/llvm/llvm-project/releases/tag/llvmorg-18.1.0>`_,
+      `18.1.1 <https://github.com/llvm/llvm-project/releases/tag/llvmorg-18.1.>`_
+    - `12.3.2 <https://developer.nvidia.com/cuda-downloads>`_\*\*
+    - **LATEST STABLE CONFIG**
+    - **LATEST STABLE CONFIG**
+  * - `19.0.0 git <https://github.com/llvm/llvm-project>`_
+    - `12.3.2 <https://developer.nvidia.com/cuda-downloads>`_
+    - \+
+    - \+
+
+:sup:`1` ``LLVM 3.x`` is no longer supported (but might still work).
+
+:sup:`2` Download the patch and unpack it into your ``LLVM distributive directory``; a few header files
+will be overwritten. You don't need to rebuild ``LLVM``.
+
+:sup:`3` Download the patch and unpack it into your ``LLVM source directory``; the ``Cuda.cpp`` file will
+be overwritten. You'll need to rebuild ``LLVM``.
+
+\*\* represents the latest supported (and recommended) configuration.
+
+In most cases, you can get a suitable version of ``LLVM+clang`` with your package manager.
+
+Failing that or having multiple versions of `LLVM`, you can
+`download a release archive <http://releases.llvm.org/>`_, build or install it, and set
+`CMAKE_PREFIX_PATH <https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html>`_ so
+``CMake`` can find it. For instance: ``-DCMAKE_PREFIX_PATH=D:\LLVM\18.1.1\dist``
+
+Usage
+============================================================
+
+To process a file, ``hipify-clang`` needs access to the same headers that would be required to compile it
+with ``clang``. For example:
+
+.. code:: shell
+
+  ./hipify-clang square.cu --cuda-path=/usr/local/cuda-12.3 -I /usr/local/cuda-12.3/samples/common/inc
+
+``hipify-clang`` arguments are given first, followed by a separator (``--``), and then the arguments you'd
+pass to Clang if you were compiling the input file. For example:
+
+.. code:: shell
+
+  ./hipify-clang cpp17.cu --cuda-path=/usr/local/cuda-12.3 -- -std=c++17
+
+``hipify-clang`` also supports the hipification of multiple files that might be specified in a single
+command line with absolute or relative paths. For example:
+
+.. code:: shell
+
+  ./hipify-clang cpp17.cu ../../square.cu /home/user/cuda/intro.cu --cuda-path=/usr/local/cuda-12.3 -- -std=c++17
+
+To use a specific version of LLVM during hipification, the ``hipify-clang`` option
+``--clang-resource-directory=`` must be specified to point to the Clang resource directory, which is the
+parent directory for the ``include`` folder that contains ``__clang_cuda_runtime_wrapper.h`` and other
+header files used during the hipification process. For example:
+
+.. code:: shell
+
+  ./hipify-clang square.cu --cuda-path=/usr/local/cuda-12.3 --clang-resource-directory=/usr/llvm/18.1.1/dist/lib/clang/18
+
+The `Clang manual for compiling CUDA <https://llvm.org/docs/CompileCudaWithLLVM.html#compiling-cuda-code>`_
+may be useful.

```

Comment:
It is taken from 6.1.1: https://github.com/ROCm/HIPIFY/blob/amd-staging/CHANGELOG.md#hipify-for-rocm-611 and is not presented in 6.1.0. 

It looks like this change is not a cherry-pick.

Reply:
Thanks @emankov. These content changes are currently live on 6.1.0/latest [ROCm docs](https://rocm.docs.amd.com/projects/HIPIFY/en/latest/hipify-clang.html).

The additions you're talking about were made in #1404 and #1391 and are in latest. #1410 keeps the same content and just converts the Markdown file into rST.

I agree there is a discrepancy with the changelog. If this content is not appropriate for 6.1.0, let's fix it here?

Reply:
The contest is newer than 6.1.0, it is targeting 6.1.x. The issue is not only in the changelog, but in the 3rd parties SW support and features, introduce later than 6.1.0.